src/components/switchComponent/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
+import {
+  Switch as ChakraSwitch,
+  SwitchProps as ChakraSwitchProps,
+} from '@chakra-ui/react';
+
+export type SwitchProps = Pick<ChakraSwitchProps, 'onChange'>;
+
+const Switch: FC<SwitchProps> = ({ onChange }) => {
+  return (
+    <>
+      <span>🚗</span>
+      &nbsp;
+      <ChakraSwitch onChange={onChange} variant="themeSwitch" />
+      &nbsp;
+      <span>🏍️</span>
+    </>
+  );
+};
```

Comment:
This is not a plain switch but a sub-component of the dev overlay. Can we move it there?